string/string.go

```diff
@@ -0,0 +1,155 @@
+package string
+
+import (
+	"strings"
+)
+
+// ToLower converts a string to lowercase.
+func ToLower(s string) string {
+	return strings.ToLower(s)
+}
+
+// ToUpper converts a string to uppercase.
+func ToUpper(s string) string {
+	return strings.ToUpper(s)
+}
+
+// Concat concatenates two strings.
+func Concat(s1, s2 string) string {
```

Comment:
May be this can the function signature can allow  the variable argument list.
func Concat(s ... string) string {
...code
}